code/modules/projectiles/ammunition/_ammunition.dm

```diff
@@ -100,9 +101,14 @@
 	bounce_away(FALSE, NONE)
 	. = ..()
 
-/obj/item/ammo_casing/proc/on_eject()
+/obj/item/ammo_casing/proc/on_eject(atom/shooter)
 	forceMove(drop_location()) //Eject casing onto ground.
-	bounce_away(TRUE)
+	pixel_x = rand(-4, 4)
+	pixel_y = rand(-4, 4)
+	pixel_z = 8 //bounce time
+	var/angle_of_movement = !isnull(shooter) ? (rand(-3000, 3000) / 100) + dir2angle(turn(shooter.dir, 180)) : rand(-3000, 3000) / 100
+	AddComponent(/datum/component/movable_physics, _horizontal_velocity = rand(400, 450) / 100, _vertical_velocity = rand(400, 450) / 100, _horizontal_friction = rand(20, 24) / 100, _z_gravity = 9.80665, _z_floor = 0, _angle_of_movement = angle_of_movement)
```

Comment:
'_z_gravity = 9.80665'
This should be a define (I am surprised it isn't already)

Reply:
I assume it's because mojave wanted it to be configurable- but you could just make more defines for that. I'll get it properly set up